content/english/python-interactive/_index.md

```diff
@@ -11,5 +11,22 @@ translationKey: "python-interactive-home"
 ---
 
 # Introduction
+Ever thought of opening your own restaurant? Well, it's your lucky day, because today, you get the chance to do one of the most important aspects - interacting with your customers and getting their orders!
 
-To start this program, open up a new Repl.it document and create a main() function definition and function call.
\ No newline at end of file
+Here's a sample interaction with the console you could have!
+![Python interactive with Console](INTERACTIVE.jpg)
+
+To do this, we're going to need to code it all out on a single Repl.it document. Follow the link below and remember to fork it before starting to code! Happy Coding!
+https://repl.it/@nuevofoundation/WearyQuestionablePostscript#main.py
```

Comment:
Can you please rename the repl.it to something more meaningful (like PythonInteractive)?